scvi/models/__init__.py

```diff
@@ -15,4 +16,6 @@
     "Classifier",
     "AutoZIVAE",
     "TOTALVI",
+    "ZINB",
```

Comment:
I'm not sure putting this here is necessary is it?, all your imports are from scvi.models.distributions